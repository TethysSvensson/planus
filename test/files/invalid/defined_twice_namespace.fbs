include "helpers/overlapping_namespace1.fbs";
include "helpers/overlapping_namespace2.fbs";

table Foo {}
