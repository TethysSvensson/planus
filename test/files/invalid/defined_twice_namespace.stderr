error: Overlapping declarations
  ┌─ test/files/invalid/defined_twice_namespace.fbs:4:1
  │
4 │ table Foo {}
  │ --------- Declaration was here
  │
  ┌─ test/files/invalid/helpers/overlapping_namespace1.fbs:1:1
  │
1 │ namespace Foo;
  │ -------------- namespace was defined here
  │
  ┌─ test/files/invalid/helpers/overlapping_namespace2.fbs:1:1
  │
1 │ namespace Foo;
  │ -------------- namespace was defined here

